source/jormungandr/jormungandr/interfaces/v1/Places.py

```diff
@@ -166,6 +166,14 @@ def output(self, key, obj):
     "address": AddressField()
 }
 
+geocode_poi = {
+    "embedded_type": Lit("poi"),
+    "quality": Lit("0"),
+    "id": AddressId,
```

Comment:
Not sure what's the difference with using `fields.String(attribute='properties.geocoding.id')`